src/SixLabors.Fonts/Tables/Cff/CffEvaluationEngine.cs

```diff
@@ -0,0 +1,747 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+namespace SixLabors.Fonts.Tables.Cff
+{
+    /// <summary>
+    /// Decodes the commands and numbers making up a Type 2 CharString. A Type 2 CharString extends on the Type 1 CharString format.
+    /// Compared to the Type 1 format, the Type 2 encoding offers smaller size and an opportunity for better rendering quality and
+    /// performance. The Type 2 charstring operators are (with one exception) a superset of the Type 1 operators.
+    /// </summary>
+    /// <remarks>
+    /// A Type 2 charstring program is a sequence of unsigned 8-bit bytes that encode numbers and operators.
+    /// The byte value specifies a operator, a number, or subsequent bytes that are to be interpreted in a specific manner
+    /// </remarks>
+    internal ref struct CffEvaluationEngine
+    {
+        private static readonly Random Random = new();
+        private float? width;
+        private int nStems;
+        private float x;
+        private float y;
+        private RefStack<float> stack;
+        private readonly ReadOnlySpan<byte> charStrings;
+        private readonly ReadOnlySpan<byte[]> globalSubrBuffers;
+        private readonly ReadOnlySpan<byte[]> localSubrBuffers;
+        private TransformingGlyphRenderer transforming;
+        private readonly int nominalWidthX;
+        private readonly int globalBias;
+        private readonly int localBias;
+        private readonly Dictionary<int, float> trans = new();
```

Comment:
I do get a build error now with Visual Studio 2019 here: 
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0573	'CffEvaluationEngine': cannot have instance property or field initializers in structs
	SixLabors.Fonts (netcoreapp3.1), SixLabors.Fonts (netstandard2.0), SixLabors.Fonts (netstandard2.1)	F:\vsprojekte\ImageSharp.Fonts\src\SixLabors.Fonts\Tables\Cff\CffEvaluationEngine.cs	35	Active
```

Works fine when I use the command line.

Reply:
Must be a langversion thing. Fixed.

Reply:
should we maybe specify the langversion explicitly instead of `latest` to avoid this?

Reply:
I thought we had via the submodules, I'll check

Reply:
Fixed